src/web/waiters/RecipeWaiter.mjs

```diff
@@ -580,29 +590,70 @@ class RecipeWaiter {
 
 
     /**
-     * Adjusts the number of ingredient columns as the width of the recipe changes.
+     * Remove all "selected" classes for op-list list items at once
+     *
+     * This hides all the checkmark icons of previously added ( selected )
+     * operations to the recipe list
      */
-    adjustWidth() {
-        const recList = document.getElementById("rec-list");
-
-        // Hide Chef icon on Bake button if the page is compressed
-        const bakeIcon = document.querySelector("#bake img");
-
-        if (recList.clientWidth < 370) {
-            // Hide Chef icon on Bake button
-            bakeIcon.style.display = "none";
-        } else {
-            bakeIcon.style.display = "inline-block";
+    clearAllSelectedClasses() {
+        const list = document.querySelectorAll(".operation.selected");
+
+        // check if any operations are selected at all to prevent errors
+        if (list.length) {
```

Comment:
Is this necessary? Doesn't `querySelectorAll` return an empty array if no items are found? Executing `forEach` on an empty array should not cause errors.